lib/commons/text/accessible-text.js

```diff
@@ -80,14 +79,14 @@ function formValueText(element) {
 		return '';
 	}
 
-	if (nodeName === 'TEXTAREA' && element.value) {
-		return element.value;
+	if (nodeName === 'TEXTAREA' && actualNode.value) {
+		return actualNode.value;
 	}
 	return '';
 }
 
-function checkDescendant(element, nodeName) {
-	var candidate = element.querySelector(nodeName.toLowerCase());
+function checkDescendant({ actualNode }, nodeName) {
+	var candidate = actualNode.querySelector(nodeName.toLowerCase());
```

Comment:
should we put a note in here to indicate that this function is intended to be used only on elements that cannot contain a shadow root?